dll/directx/wine/d3dcompiler_43/d3dcompiler_private.h

```diff
@@ -38,6 +38,14 @@
 #include <wine/list.h>
 #include <wine/rbtree.h>
 
+#if defined(_UNITY_BUILD_ENABLED_)
+WINE_FORWARD_DECLARE_DEBUG_CHANNEL(asmshader);
+WINE_FORWARD_DECLARE_DEBUG_CHANNEL(d3dcompiler);
+WINE_FORWARD_DECLARE_DEBUG_CHANNEL(bytecodewriter);
+WINE_FORWARD_DECLARE_DEBUG_CHANNEL(hlsl_parser);
+WINE_FORWARD_DECLARE_DEBUG_CHANNEL(parsed_shader);
+#endif
```

Comment:
If something is missing from this list, it will issue a compiler error?

Reply:
Yes.

Reply:
Great.